src/ai/backend/manager/api/exceptions.py

```diff
@@ -173,6 +173,11 @@ class AuthorizationFailed(BackendError, web.HTTPUnauthorized):
     error_title = "Credential/signature mismatch."
 
 
+class PasswordExpired(BackendError, web.HTTPUnauthorized):
+    error_type = "https://api.backend.ai/probs/password-expired"
+    error_title = "Should update password."
```

Comment:
It would be helpful if the error message for PasswordExpired is more specific, such as "Password has exceeded the maxage for change."